src/deploy.sh

```diff
@@ -509,10 +516,70 @@ function kernel_deploy()
   fi
 }
 
+# Deploy a package
+#
+# @flag How to display commands. See `cmd_manager` inside `src/kwlib.sh`
+# @pkg_type The type of package
+# @target VM, local or remote?
+# @remote_raw Address of remote target
+#
+function pkg_deploy()
+{
+  local flag="$1"
+  local pkg_type="$2"
+  local target="$3"
+  local remote_raw="$4"
+  local package_path
+  local target_distro
+  local host
+  local port
+
+  package_path="$(tail -1 < "$KW_LIB_DIR/PKG_INFO")"
+  package_type="$(head -1 | "$KW_LIB_DIR/PKG_INFO")"
+  package_name="$(basename "$package_path")"
```

Comment:
Could you add a few lines explaining what's going on here? I think you can also document it a little bit in the function doc and in the commit message.